src/nbd.c

```diff
@@ -753,6 +754,13 @@ static gboolean finish_configure(struct RaucNBDContext *ctx, struct RaucNBDTrans
 		ctx->url = g_strdup(effective_url);
 	}
 
+	code = curl_easy_getinfo(xfer->easy, CURLINFO_HTTP_VERSION, &http_version);
+	if (code == CURLE_OK) {
+		if (http_version == CURL_HTTP_VERSION_1_0 ||
```

Comment:
Does HTTP 1.0 even have range requests? Would this code work without?

Reply:
Server may respond strangely, so I'd include 1.0 just in case.